docs/source/use_cases/run_familiar_tools/bioconductor.rst

```diff
@@ -36,6 +36,12 @@ If you want to deploy a different docker container, such as the one from :doc:`/
 2. Click on *More* to display the additional form fields.
 3. In field *Custom docker image* paste in the docker image path, such as ``gcr.io/bioc_2015/devel_sequencing`` or ``b.gcr.io/isb-cgc-public-docker-images/r-examples``.
 
+Temporarily switch to a machine with more RAM
```

Comment:
Why is this listed as "Temporarily"?
Should this just say:

Change your virtual machine type (number of cores, amount of memory)